src/cata_imgui.cpp

```diff
@@ -36,8 +36,39 @@ struct pairs {
     short BG;
 };
 
+ImVec4 impalette[256] = {};
 std::array<RGBTuple, color_loader<RGBTuple>::COLOR_NAMES_COUNT> rgbPalette;
-std::array<pairs, 100> colorpairs;   //storage for pair'ed colored
+std::array<pairs, 100> colorpairs;   //storage for paired colors
+
+static ImVec4 compute_color( uint8_t index )
```

Comment:
Can argument be const uint8_t here?

Reply:
I don’t see the point. It’s copied, so even if this function modified it the caller wouldn’t care.

Reply:
The point is communicating intent about what can and should be done with a variable, and what cannot and should not.

Reply:
Again, this is a simple u8. The caller gives us a copy of it. If we modify it, we modify only our own private copy. The caller’s value cannot be changed, whether we put `const` or not. Thus, adding `const` doesn't change the contract.